docs/administration/configuration/configuring-external-access/index.md

```diff
@@ -0,0 +1,136 @@
+
+You can also specify additional Ingresses with the `ingress.extra` Helm value.
+Each additional Ingress can be configured in the same way as the default one, with the addition of `ingress.extra[].name`.
+
+As an example, this is how you would configure Traefik by creating two Ingresses, one to handle HTTPS traffic and the other to redirect HTTP requests to HTTPS.
+
+```{ .yaml .copy-to-clipboard }
+ingress:
+  enabled: true
+  tls:
+    self_signed_cert: false
+    secret_name: reana-tls-secret
+    hosts:
+      - reana.cern.ch
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/router.entrypoints: https
+    traefik.ingress.kubernetes.io/router.tls: "true"
+  extra:
+    - name: http
+      annotations:
+        kubernetes.io/ingress.class: traefik
+        traefik.ingress.kubernetes.io/router.entrypoints: http
+        traefik.ingress.kubernetes.io/router.middlewares: kube-system-redirect-scheme@kubernetescrd
+```
+
+## Manual setup
+
+If you would like to manually manage Ingresses, you should set `ingress.enabled` to false to disable the automatic creation of Ingresses.
+In the same way, you can disable the internal Traefik instance by setting `traefik.enabled` to false.
+
+Your custom Ingress must redirect all the requests to `/api` and `/oauth` to the `<helm-release-prefix>-server` Service, while all the rest should be sent to `<helm-release-prefix>-ui`
+You can find an example of custom Ingress below.
+
+??? note "Example of custom Ingress"
+
+    ```{ .yaml .copy-to-clipboard }
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      name: reana-ingress
+      annotations:
+        kubernetes.io/ingress.class: traefik
+        traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
+    spec:
+      tls:
+        - secretName: <tls-secret>
+      rules:
+        - http:
+            paths:
+            - path: /api
+              pathType: Prefix
+              backend:
+                service:
+                  name: <helm-release-prefix>-server
+                  port:
+                    number: 80
+            - path: /oauth
+              pathType: Prefix
+              backend:
+                service:
+                  name: <helm-release-prefix>-server
+                  port:
+                    number: 80
+            - path: /
+              pathType: Prefix
+              backend:
+                service:
+                  name: <helm-release-prefix>-ui
+                  port:
+                    number: 80
+          host: <hostname>
+    ```
+
+## Reverse proxy
+
+You might be interested in placing a (reverse) proxy in front of REANA, but some additional configuration might be needed to make REANA work correctly.
+
+Let's consider a setup in which a generic proxy handles connections to REANA, like so:
```

Comment:
Let us consider...  (slightly more neutral that contracted "let's")


... handles connections to REANA: